src/appmixer/utils/csv/AddRows/AddRows.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const CSVProcessor = require('../CSVProcessor');
+
+module.exports = {
+
+    async receive(context) {
+
+        const { withHeaders } = context.properties;
```

Comment:
it's not used here (there is no definition in the compoentn.json either), I think we can remove `withHeaders` completely

Reply:
btw, why is `withHeaders` is not implemented here?